extensions/datasource/deployment-spi/src/main/java/io/quarkus/datasource/deployment/spi/DevServicesDatasourceProvider.java

```diff
@@ -9,7 +9,7 @@
 public interface DevServicesDatasourceProvider {
 
     RunningDevServicesDatasource startDatabase(Optional<String> username, Optional<String> password,
-            Optional<String> datasourceName,
+            String datasourceName,
```

Comment:
This (and maybe other changes) will break backwards compatibility with some Quarkiverse extensions such as the one providing the SQLite JDBC driver.

Just mentioning in case you weren't aware, but I don't personally think it's a big issue.

Reply:
Yeah. It might not be worth breaking it. I'll revisit it.